cmd/create/cluster/cmd.go

```diff
@@ -3360,8 +3365,9 @@ func validateAvailabilityZones(multiAZ bool, availabilityZones []string, awsClie
 	}
 	for _, az := range availabilityZones {
 		if !helper.Contains(regionAvailabilityZones, az) {
+			reg, _ := awsClient.GetRegion()
```

Comment:
We should check the error returned from the function, by changing the `GetRegion` to get the region from the `cfg` stored in the `awsClient` struct, the function will return only a string.